src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -42,6 +42,9 @@
     "If needed, use the --role argument to explicitly create a role assignment."
 )
 
+NAME_DEPRECATION_WARNING = \
+    "'name' property in the output is deprecated and will be removed in the future. Use 'appId' instead."
```

Comment:
Shall we tell users that 'name' property in the output has changed from identifierUris to appId. If they want to get identifierUris, use `az ad app show` instead?

Reply:
This information is already conveyed in the history. I prefer not to mentioned it here as 

- it is used as an opaque box 
- it will make the already-overly-crowded warning messages even more crowded